frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -857,6 +815,27 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                 return Object.keys(snapshotsByWindowId)
             },
         ],
+
+        createExportJSON: [
+            (s) => [s.snapshots, s.sessionPlayerMetaData],
+            (
+                snapshots,
+                sessionPlayerMetaData
+            ): ((exportUntransformedMobileSnapshotData: boolean) => ExportedSessionRecordingFileV2) => {
+                return (exportUntransformedMobileSnapshotData: boolean) => ({
+                    version: '2023-04-28',
+                    data: {
+                        id: sessionPlayerMetaData?.id ?? '',
+                        person: sessionPlayerMetaData?.person,
+                        snapshots: snapshots,
+                        // TODO: What about this?!
+                        // snapshots:  exportUntransformedMobileSnapshotData
+                        //     ? sessionPlayerSnapshotData?.untransformed_snapshots || []
+                        //     : sessionPlayerSnapshotData?.snapshots || [],
```

Comment:
definitely still need that, if that's the question, so we can export mobile replay for our testing

Reply:
Yeah it will make a come back